services/spar/test-integration/Util/Scim.hs

```diff
@@ -381,6 +396,9 @@ class IsUser u where
     maybeName :: Maybe (u -> Maybe Name)
     maybeTenant :: Maybe (u -> Maybe SAML.Issuer)
     maybeSubject :: Maybe (u -> Maybe SAML.NameID)
+    -- | Not all types have a 'SAML.NameID' but some of them have a subject ID anyway, just
+    -- not in a structured form.
+    maybeSubjectRaw :: Maybe (u -> Maybe Text)
```

Comment:
would be nice to give two example types here, like `UserRef` and `ScimUser` (or whathever it's called)?